plugins/services/src/js/constants/VolumeDefinitions.js

```diff
@@ -0,0 +1,29 @@
+var VolumeDefinitions = {
+  DSS: {
+    name: "DC/OS Storage Service",
+    type: "DC/OS",
+    description: "A locally persistent volume pre-created by the operator according to a storage profile."
+  },
+  PERSISTENT: {
+    name: "Local Persistent Volume",
+    type: "Persistent",
+    description: "A locally persistent volume based upon the physical disks installed in the agent which has the capabilities of a single disk."
+  },
+  EXTERNAL: {
+    name: "External Persistent Volume",
+    type: "External",
+    description: "A REX-Ray backed volume that is mounted on the agent from network attached storage."
+  },
+  HOST: {
+    name: "Host Persistent Volume",
```

Comment:
It is just Host Volume,
A Host volume is basically just a directory which is getting mounted into the docker container.

Reply:
It's addressed. Thank you.